### DISCUSSION TITLE
Message Service

### DISCUSSION THREAD

**Discussion by Keltanis** [Megabit/Blazorise#3629] [Q&A]
Currently have a component rendered inside of a modal. The Modal is full height. When I click on a button to show my message, that is also rendered at full height. Is it possible to adjust the height of the Message or Alert by setting it before it displays?

Using the code found in Docs:
```
<Button Color="Color.Primary" Clicked="@ShowInfoMessage">Say hi!</Button>
@code{
    [Inject] IMessageService MessageService { get; set; }

    Task ShowInfoMessage()
    {
        // MessageService.Height or something would be awesome. Actually any way would be nice.
        return MessageService.Info( "This is a simple info message!", "Hello" );
    }
}
```
Before Message Service is called:
![BeforeMessage](https://user-images.githubusercontent.com/42776871/159815840-c7bd2d52-862c-4700-afc0-f2fa99ecb2c0.jpg)

Message popup takes 100% of modal height:
![WithMessage](https://user-images.githubusercontent.com/42776871/159815849-d75a088e-70ad-4c6e-943e-a99a35439786.jpg)

**Comment by stsrki** [comment]
Can you tell me where did you put the `<MessageAlert />` component? Is it in `App.razor` or somewhere else?

**Comment by Keltanis** [comment]
In the app.razor as suggested. 




Thursday, March 24, 2022, 9:33 AM -0600 from ***@***.***  ***@***.***>:

**Comment by Keltanis** [comment]
Also, found out we are using Bootstrap 4.*

Maybe that?




Thursday, March 24, 2022, 9:35 AM -0600 from Brent Ricks  ***@***.***>:

**Comment by stsrki** [comment]
That shouldn't be that. Both modals should be completely separated. in the DOM so not sure why they would be affected by each other. Can you try showing the message service modal from somewhere else, not from the "Choose a file" modal?

**Comment by Keltanis** [comment]
I will try a little later today and get back to you. 




Thursday, March 24, 2022, 9:52 AM -0600 from ***@***.***  ***@***.***>:

**Comment by Keltanis** [comment]
Just verified that outside of the modal, does the same thing. 




Thursday, March 24, 2022, 9:52 AM -0600 from ***@***.***  ***@***.***>:

**Reply by stsrki** [reply]
I'm lost now 😅

Any chance you could provide a minimal project to reproduce it on our side? Btw, is this an ABP project?

**Comment by Keltanis** [comment]
Unfortunately, no. However, your comments makes me believe it all s being overridden somewhere in this massive project I’m working on. 

The project is a MVC/Blazor WASM monster with many cooks. 

For now, the Blazorise Alerts seem to work as intended and I can use that instead, I believe. Thank you for narrowing down my possibilities. 



Friday, March 25, 2022, 12:55 AM -0600 from ***@***.***  ***@***.***>:
